doc/changes/latest.rst

```diff
@@ -13,6 +13,8 @@ HIGHLIGHTS
 Fixes
 -----
 
+- :bdg-dark:`Code` Prevent range overrun in TFCE (:gh:`5179` by `Lee Newberg`_).
+
```

Comment:
I wonder if the changelog should be updated further to mention that the TFCE procedure has now an upper and lower bound in the number of thresholds it will test.

Reply:
I just modified the first commit to update the changelog for that, and forced pushed.

Reply:
yeah but that removed the last commit I added.

Reply:
in general there is no need to rewrite older commits of a pull request because we will do a squash-merge of all the commit of the PR: so just add new changes and keep pushing

will get back my commit from the dead

Reply:
no worries: I already repushed it
so if you do a git pull on your branch you should have it